src/Templates/AllWidget/AllWidget.js

```diff
@@ -571,15 +575,14 @@ export class AllWidget {
       min_y,
       max_y
     );
-
     // Get camera placement parameters from config
     let coordinates = this.extent.center();
     if (
       this.config['camera']['position']['x'] &&
       this.config['camera']['position']['y']
-    ) {
+    ) {      
```

Comment:
Coding style ok ?

Reply:
My eslint made this coding style when formatting the file using it ... I suppose this is ok then ?